src/OrchardCore.Modules/OrchardCore.OpenId/Manifest.cs

```diff
@@ -2,25 +2,25 @@
 using OrchardCore.OpenId;
 
 [assembly: Module(
-    Name = "OpenID",
+    Name = "OpenID Connect",
     Author = ManifestConstants.OrchardCoreTeam,
     Website = ManifestConstants.OrchardCoreWebsite,
     Version = ManifestConstants.OrchardCoreVersion
 )]
 
 [assembly: Feature(
     Id = OpenIdConstants.Features.Core,
-    Name = "OpenID Core Components",
+    Name = "Core OpenID Connect Services",
```

Comment:
IMHO, it would be better if all the features started with `OpenID Connect` (here, `OpenID Connect Core Services`)

Reply:
done